how-to/workspace-platform-starter/client/src/modules/composite/pages/actions.ts

```diff
@@ -64,16 +63,15 @@ export class PageActions implements Actions {
 							const targetWindow = platform.Browser.wrapSync(targetWindowIdentity);
 							await targetWindow.addPage(page);
 							await targetWindow.setActivePage(pageId);
-						} else {
+						} else if (this._helpers?.launchPage !== undefined) {
```

Comment:
Use !isEmpty

Reply:
Typescript complains that the function could still be undefined. And we are really just checking to see if the function is set which would be undefined or true (so skips a function call). So left as is.

Reply:
Just change it to `if (this._helpers?.launchPage)` then, the new lint rules will fail it otherwise

Reply:
Didn't know about the new linting rule. Updated.